plugins/system/skeletonkey/media/js/backend.js

```diff
@@ -40,7 +40,7 @@ const initSkeletonKey = () => {
 
 		// Create the icon for the login button
 		const elIcon = document.createElement('span');
-		elIcon.classList.add('fa', 'fa-sign-in-alt', 'me-1');
+		elIcon.classList.add('icon-external-link-alt', 'me-1');
```

Comment:
This needs changing to `'fa', 'fa-external-link-alt'`. Joomla 4 and later versions use FontAwesome. The `icon-` class prefix is an alias to the `fa-` prefix. We should be using FontAwesome directly.